spec/abstractops.html

```diff
@@ -1845,4 +1845,40 @@ <h1>PreparePartialTemporalFields ( _fields_, _fieldNames_ )</h1>
       1. Return _result_.
     </emu-alg>
   </emu-clause>
+   <emu-clause id="sec-temporal-getdifferencesettings" type="abstract operation">
+    <h1>
+      GetDifferenceSettings (
+        _operation_: ~since~ or ~until~,
+        _options_: an ECMAScript language value,
+        _disallowedUnits_ : a List of String,
+        _fallbackSmallestUnit_ : a String,
+        _smallestLargestDefaultUnit_ : a String,
+      ): either a normal completion containing a Record or an abrupt completion
+    </h1>
+    <dl class="header">
+      <dt>description</dt>
+      <dd>It reads unit and rounding options needed by difference operations.</dd>
+    </dl>
+    <emu-alg>
+      1. Set _options_ to ? GetOptionsObject(_options_).
+      1. Let _smallestUnit_ be ? ToSmallestTemporalUnit(_options_, _disallowedUnits_, _fallbackSmallestUnit_).
+      1. Let _defaultLargestUnit_ be ! LargerOfTwoTemporalUnits(_smallestLargestDefaultUnit_, _smallestUnit_).
+      1. Let _largestUnit_ be ? ToLargestTemporalUnit(_options_, _disallowedUnits_, *"auto"*, _defaultLargestUnit_).
+      1. Perform ? ValidateTemporalUnitRange(_largestUnit_, _smallestUnit_).
+      1. Let _roundingMode_ be ? ToTemporalRoundingMode(_options_, *"trunc"*).
+      1. If _operation_ is ~since~, then
+        1. Set _roundingMode_ to ! NegateTemporalRoundingMode(_roundingMode_).
+      1. Let _maximum_ be ! MaximumTemporalDurationRoundingIncrement(_smallestUnit_).
+      1. Let _roundingIncrement_ be ? ToTemporalRoundingIncrement(_options_, _maximum_, *false*).
+      1. Let _untilOptions_ be ? MergeLargestUnitOption(_options_, _largestUnit_).
```

Comment:
Not all cases were calling MergeLargestUnitOption before, and that's very much observable.

Reply:
yea, I was debatting myself should I not include the creation of untilOptions here. Including it here surely make it Normative change. I will remove that.